lib/diagnostic.js

```diff
@@ -39,24 +51,48 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = new Object{}
+  pet.name =  "Jack",
+  pet.species =  "dog",
+  pet.breed =  "chocolate lab",
+  pet.vocalization = "barks",
+  pet.vocalize = function()
```

Comment:
Make sure this function actually does something! In this case, you want it to return the `vocalization` property you defined earlier.